srcpkgs/python3-platformdirs/template

```diff
@@ -13,8 +13,10 @@ license="MIT"
 homepage="https://github.com/platformdirs/platformdirs"
 changelog="https://raw.githubusercontent.com/platformdirs/platformdirs/main/CHANGES.rst"
 distfiles="${PYPI_SITE}/p/platformdirs/platformdirs-${version}.tar.gz"
-checksum=15b056538719b1c94bdaccb29e5f81879c7f7f0f4a153f46086d155dffcd4f0f
+checksum=58c8abb07dcb441e6ee4b11d8df0ac856038f944ab98b7be6b27b2a3c7feef19
+make_check_pre="env PYTHONPATH=src"
 
 post_install() {
+	# TODO: reanme to LICENSE for later versions
```

Comment:
why?

Reply:
https://github.com/platformdirs/platformdirs/commit/2b59bccaedf99c913b6ecc6a7c8a50c97ceb11c8
https://github.com/platformdirs/platformdirs/commit/ed4ecdbbf5db987be89644654035eb05c0769ed7

Reply:
ok, the vlicense should fail on its own in the next update, which should be enough of a notification I think.